blacklistfilter/evaluator/split_evidence/fields.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _UR_FIELDS_H_
```

Comment:
fields.h file should not be versioned in git repo, it should be generated automatically by Makefile